config/initializers/helo.rb

```diff
@@ -0,0 +1,17 @@
+if ENV['HELO_ENABLED'] == 'enabled'
+  ActiveSupport.on_load(:action_mailer) do
+    module Helo
+      class SMTP < ::Mail::SMTP; end
```

Comment:
Simplement sur le style. Une syntaxe que je trouve plus élégante et plus lisible : 

```suggestion
      SMTP  = Class.new(::Mail::SMTP)
```

documentation : https://devdocs.io/ruby~3/class#method-c-new

Reply:
Ah oui, c'est ce que j'utilisais pour faire des Struct, mais je savais pas qu'on pouvait le faire aussi pour des classes.

Cela dit dans le cas présent je suis plutôt pour utiliser la syntaxe la plus habituelle, pour la moindre surprise. Mais merci pour la découverte !